app/javascript/components/widgets/guests/index.jsx

```diff
@@ -0,0 +1,7 @@
+import Carousel from './carousel';
+import Panel from './panel';
+
+export default {
+  Carousel,
```

Comment:
when we are exporting we can do it like 
```
export default {
  CarouselItem: GuestsCarouselItem,
  Panel: GuestsPanel,
}
```

Reply:
I don't think this intermediary naming has an impact on the dev tools. Was there another reason to prefer `import GuestsCarouselItem from './carousel'` over `import Carousel from './carousel'`

Reply:
i like the imports to match the names they are exporting. I think it reduces the overhead when trying to understand files like this.